### PR TITLE
onboarding: Create onboarding guide for General Members

### PR DESCRIPTION
**Pull request by justaugustus**
Continues https://github.com/todogroup/governance/pull/165.

- onboarding: Init directory
- onboarding: Create onboarding guide for General Members

  This commit splits the General Member onboarding issue template
  into two documents:

  - an issue template for TODO Group PgMs & Steering Committee to
    operationalize access to various TODO Group resources
  - an onboarding guide for new TODO Group members to review to
    ensure they're aware of the various communications channels
    / resources available to them

Signed-off-by: Stephen Augustus <foo@auggie.dev>

### CODE REVIEW

File: onboarding/general-member.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+An issue should have been created to process your company's membership (via this [template](/.github/ISSUE_TEMPLATE/onboarding-general-member.md)).
+
+If you run into any issues during the onboarding process, please reach out to the TODO Group Program Manager ([@anajsana][todo-pm]).
+
+- [Communication channels](#communication-channels)
+  - [Registration required](#registration-required)
+  - [Social media](#social-media)
+  - [GitHub](#github)
+  - [Private channels](#private-channels)
```
Comment (anajsana) [inline onboarding/general-member.md:17]: can we call it "General Member" Channels?

File: onboarding/general-member.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- [Meetings and Calls](#meetings-and-calls)
+  - [Public calls](#public-calls)
+  - [Private calls](#private-calls)
```
Comment (anajsana) [inline onboarding/general-member.md:20]: Can we call it "General Member" calls? 🙂

File: onboarding/general-member.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+<!--
+TODO: Publish this info on the website to de-duplicate content
+-->
+
+### Registration required
+
+- [TODO Group Slack](https://slack.todogroup.org/)
+- [Community Mailing List](https://docs.google.com/forms/d/e/1FAIpQLSeU0YGM_IJ6gY8E5IIiwXKD_FZi3kAVc4E9_-3dtTDyKMSjdA/viewform)
+- [OSPO Newsletter](https://www.getrevue.co/profile/osponews)
```
Comment (anajsana) [inline onboarding/general-member.md:33]: Actually, everyone has access to the newsletter issues without subscribing to the newsletter (e.g see https://www.getrevue.co/profile/osponews/issues/ospo-news-from-the-todo-group-issue-7-959516). However, if people want to get notifications when the newsletter is sent and have it in their emails, then OSPONews subscription is recommended 👍

File: onboarding/general-member.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- [OSPOlogy](https://github.com/todogroup/ospology/discussions) (OSPO Forum Discussions)
```
Comment (anajsana) [inline onboarding/general-member.md:34]: I'll call it "OSPO Forum" and people don't have to register to have access to the content (however, if they want to comment on a discussion, they might need to sign in in GitHub)

File: onboarding/general-member.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- [Resources and initiatives](#resources-and-initiatives)
+
+## Communication channels
+
+<!--
+TODO: Publish this info on the website to de-duplicate content
+-->
+
+### Registration required
+
```
Comment (anajsana) [inline onboarding/general-member.md:30]: I'd add an intro description. Something like:

"Public communication channels that need registration wihtin third-party platforms to either see content or interact with content (Slack, LF Community Platform or GitHub)"

File: onboarding/general-member.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+### Private channels
+
+- [General Member mailing list](https://groups.google.com/g/private-todogroup)
+- General Member Slack channel: [#private](https://thetodogroup.slack.com/archives/G5TBGTX29)
+
+## Meetings and Calls
+
+### Public calls
+
```
Comment (anajsana) [inline onboarding/general-member.md:53]: Clarify that needs RSVP prior the meeting starts 👍

File: onboarding/general-member.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+- General Members Touchpoint (an invite is sent once you are added to the TODO private mailing list): Bi-weekly informal meetings on Fridays
+- General Members AMA session (an invite is sent once you are added to the TODO private mailing list): Quarterly on mondays
+
+## Resources and initiatives
+
+- Review the existing GitHub repos in the TODO Group org's [`README.md`](https://github.com/todogroup)
+- Read TODO Group [FAQ](https://todogroup.org/faq/)
+- Read the [TODO Charter](https://github.com/todogroup/governance/blob/main/CHARTER.adoc)
+
```
Comment (anajsana) [inline onboarding/general-member.md:67]: I would like to add an extra section called "First-time contributions" to at point new members to contribution guidelines and repos where they can start adding their first contributions to TODO:

## Contributing Guidelines

* [OSPO Case Study](https://todogroup.org/guides/casestudies/todo-contribution-guidelines/)
* [TODO Guides](https://todogroup.org/guides/todo-guides-contribution-guidelines/)
* [Whitepapers](https://todogroup.org/guides/whitepaper-guidelines/)
* [OSPOlogy Meetings](https://github.com/todogroup/ospology/tree/main/meetings#what-are-ospology-monthly-meetings)
* [OSPO Survey](https://github.com/todogroup/osposurvey/tree/master/2022#how-to-contribute)

**Review by anajsana — CHANGES_REQUESTED** [review]
This looks great! Know it gets much more organized IMHO, so thanks a lot 😊 I've left some suggestions to `general-member.md`

**Comment by justaugustus** [comment]
> I've left some suggestions to `general-member.md`

@anajsana -- Awesome suggestions! I've integrated them as https://github.com/todogroup/governance/pull/166/commits/840c99d3fd624a5c52a8e15fd42a80a7a6172e4c.

**Comment by anajsana** [comment]
I think we can merge this. LGTM👏

**Pull request merged**